app/assets/stylesheets/recipe_finder.css.scss

```diff
@@ -1,45 +1,37 @@
-$color-light-yellow: #FFF888;
-$color-dirt-yellow: #E8AC06;
-$color-orange: #FF8508;
-$color-red: #E83505;
-$color-raisin: #5C0B22;
-$texture-picnic-pattern: linear-gradient(
-      rgba(0, 0, 0, 0.2), 
-      rgba(0, 0, 0, 0.2)
-    ), image-url('lunch_1x.jpg');
-$texture-food-pattern: linear-gradient(
-      rgba(0, 0, 0, 0.1), 
-      rgba(0, 0, 0, 0.1)
-), image-url('food.png');
-$texture-marble-pattern: image-url('congruent_outline.png');
-$texture-wood-pattern: linear-gradient(
-      rgba(0.2, 0.2, 0.2, 0.1), 
-      rgba(0.2, 0.2, 0.2, 0.1)
-), image-url('retina_wood.png');
-
-// $font4: 'Yellowtail', cursive;
-// $font4: 'Bad Script', cursive;
-// $font4: 'Marck Script', cursive;
-$font-items: 'Bitter', serif;
-// $font4: 'Lobster', cursive;
-// $font4: 'Special Elite', cursive;
-$font-title: 'Satisfy', cursive;
-// $font4: 'Damion', cursive;
-$font-footer: 'Architects Daughter', cursive;
-// $font4: 'Cookie', cursive;
+@import "variables";
 
 .recipe-finder-body {
   // background: white;
-  background: $texture-food-pattern;
+  background: transparent;
+  color: $color-airbnb-font;
+  font-family: $font-airbnb;
   // height: 580px;
-  border-radius: 10px;
+  // border-radius: 10px;
   padding: 20px;
-  box-shadow: 8px 8px 25px #111;
+  // box-shadow: 8px 8px 25px #111;
+}
+
+.ui-slider-range.ui-widget-header.ui-corner-all {
+  background: $color-airbnb-red;
```

Comment:
you probably don't want to call a variable "red" our "white" or anything, because one purpose of those variables (on top of keeping your code DRY) is to give you the ability to change change at anytime. What if you want to turn that red into yellow, would you just change one line in the variables file, or would you want to change all the variable names as well, on every css file where it's been used?
--> The css names in general should be **context-specific** (.main-background, .slider, .new-recipe-title for classes, $primary-color, $border-color for variables) and not **style-specific** (classes like .red, .bold-text, and .margin-10px don't make any sense)